client/my-sites/upgrades/domain-management/edit/mapped-domain.jsx

```diff
@@ -110,10 +111,11 @@ const MappedDomain = React.createClass( {
 
 		return (
 			<VerticalNavItem path={ path }>
-				{ this.translate( 'DNS Records' ) }
+				{ this.props.translate( 'DNS Records' ) }
 			</VerticalNavItem>
 		);
 	}
 } );
 
-module.exports = MappedDomain;
+export { MappedDomain };
```

Comment:
Thanks for the link, @gwwar - now I understand the reason better. My biggest beef with this is that it adds an export that is essentially only useful for testing (and should actually not be used for anything else). But I guess we've got to make compromises sometimes :)
Thanks again! 🙇